lib/aws/generated/e_m_rcontainers.ex

```diff
@@ -0,0 +1,589 @@
+# WARNING: DO NOT EDIT, AUTO-GENERATED CODE!
+# See https://github.com/aws-beam/aws-codegen for more details.
+
+defmodule AWS.EMRcontainers do
```

Comment:
Maybe including EMR as `names_to_capitalize` in aws-codegen's `Name` module can solve the problem of the filename here.